decentralized_identity_proof_DNS-TXT.md

```diff
@@ -91,6 +91,27 @@ This will unfortunately be an unresolved issue if we use DNS, as DNS has no arch
 #### DNS Spoofing
 [DNS spoofing](https://en.wikipedia.org/wiki/DNS_spoofing) is partially mitigated by using a well known DNS resolver such as Google or Cloudflare to retrieve the TXT records. However using DNSSEC can fully resolve the issue.
 
+## Using DNS to verify the identity of a signer
+When signing a certificate, one can add it's public key into the `proof.publicKey` field, so that OA can automatically verify that the identity of the signer:
+```json
+"proof": {
+  "type": "EcdsaSecp256k1Signature2019",
+  "created": "2020-04-26T21:41:34.663Z",
+  "proofPurpose": "assertionMethod",
+  "publicKey": "0x44E682d207bcDDDAD0Bb3a650cCb9de0911B9D3A#owner",
```

Comment:
Either need to drop the `#owner` fragment if noting the `publicKey` directly or need to make this a fully compliant DID pointing to the `publicKey` in the DIDDocument: `did:ethr:0x44E682d207bcDDDAD0Bb3a650cCb9de0911B9D3A#owner`

If using the property name `publicKey` instead of `verificationMethod` I would suggest just dropping the `#owner` fragment.